awx/main/tests/functional/test_credential.py

```diff
@@ -206,10 +204,11 @@ def test_vault_validation(organization, inputs, valid):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('become_method, valid', list(zip(
-    dict(V1Credential.FIELDS['become_method'].choices).keys(),
-    itertools.repeat(True)
-)) + [('invalid-choice', False)])
+@pytest.mark.parametrize('become_method, valid', [
+    ('', True),
```

Comment:
Does blank `become_method` mean that it should not use any `become_method`?

Reply:
The behavior in that case is supposed to be that of the Ansible CLI itself, to use `sudo`.